#Expyriment 1.0.0 (Python 3.13.7), .xpe-file, coding: cp1252
#date: Sat Oct 11 2025 20:53:25
#sha1: e16f85
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='DESKTOP-GNBUSH4', release='11', version='10.0.26100', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
229596,design,log,
#design: #exp: Blindspot
#design: block_cnt,block_id,trial_cnt,trial_id
229596,design,logged,
229596,Experiment,started
233073,Keyboard,received,1073742048,wait
233082,Stimulus,presented,1
233088,Stimulus,presented,2
236873,Keyboard,received,1073741905,wait
236873,stderr,received,'Exception ignored in: <function Visual.__del__ at 0x000001E066142A20>'
236873,stderr,received,'Traceback (most recent call last):'
236873,stderr,received,'  File "C:\\Users\\lucie\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\expyriment\\stimuli\\_visual.py", line 266, in __del__'
236874,stderr,received,'    if self._compression_filename is not None:'
236874,stderr,received,"AttributeError: 'Circle' object has no attribute '_compression_filename'"
236890,stderr,received,'Traceback (most recent call last):'
236891,stderr,received,'  File "C:\\Users\\lucie\\Documents\\CogSup\\Programming\\Assignments\\Week-5\\Exercises\\blindspot.py", line 65, in <module>\n    while True :\n    ^^^^^^^^^^^'
236891,stderr,received,'  File "C:\\Users\\lucie\\Documents\\CogSup\\Programming\\Assignments\\Week-5\\Exercises\\blindspot.py", line 33, in run_trial\n    stimuli.Circle(r = r, pos=(x, y - 20))\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^'
236891,stderr,received,"TypeError: Circle.__init__() got an unexpected keyword argument 'r'"
